src/Console/Commands/MakeWidget.php

```diff
@@ -29,16 +29,16 @@ public function handle(): void
     {
         $name = $this->argument('name');
         $className = Str::studly($name);
-        $snakeCaseClassName = Str::snake($className);
+        $kebabCaseClassName = Str::kebab($className);
 
         $stub = $this->getStubContents();
-        $stub = str_replace(['{{class}}', '{{snake_case class}}'], [$className, $snakeCaseClassName], $stub);
+        $stub = str_replace(['{{class}}', '{{snake_case class}}'], [$className, $kebabCaseClassName], $stub);
```

Comment:
This is confusing, text is snake_case class, and it gets replaced with kebab-case